x-pack/plugins/security_solution/server/usage/detections/detections_helpers.ts

```diff
@@ -301,3 +483,115 @@ export const getMlJobMetrics = async (
 
   return [];
 };
+
+export const getDetectionRuleMetrics = async (
+  index: string,
+  esClient: ElasticsearchClient,
+  savedObjectClient: SavedObjectsClientContract
+): Promise<DetectionRuleAdoption> => {
+  let rulesUsage: DetectionRulesTypeUsage = initialDetectionRulesUsage;
+  const ruleSearchOptions: RuleSearchParams = {
+    body: { query: { bool: { filter: { term: { 'alert.alertTypeId': SIGNALS_ID } } } } },
+    filterPath: [],
+    ignoreUnavailable: true,
+    index,
+    size: 10_000,
+  };
+
+  let detectionAlertsResp: AlertsAggregationResponse | undefined;
+
+  try {
+    const { body: ruleResults } = await esClient.search<RuleSearchResult>(ruleSearchOptions);
+
+    // @ts-expect-error `SearchResponse['hits']['total']` incorrectly expects `number` type instead of `{ value: number }`.
+    ({ body: detectionAlertsResp } = await esClient.search({
+      index: '.siem-signals-pjhampton-default', // TODO:PH pass in siem-sigs index
```

Comment:
Is this index correct?

Reply:
Thanks! Updated in https://github.com/elastic/kibana/pull/95659/commits/40753aab3c74ce8f729ec1c5a127161d3b2c39c4